app/templates/public/schedule.hbs

```diff
@@ -1 +1,7 @@
-{{outlet}}
+
+{{#if outlet}}
+  {{outlet}}
+{{else}}
+  <h4>*Schedule for the event is currently not available</h4>
```

Comment:
For the helper text use `<div class="ui disabled header">` instead of the h4 tag.

Reply:
Ok right on it